### ISSUE TITLE
support env variables for  `-t` and `-H`

### ISSUE THREAD

**Issue by shenwei356** [shenwei356/csvtk#39]

**Comment by shenwei356** [comment]
Available in [v0.13.0-dev2](https://github.com/shenwei356/csvtk/releases/tag/v0.13.0-dev2) or later versions. Examples:

```
$ cat digitals.tsv 
4       5       6
1       2       3
7       8       0
8       1,000   4

$ cat digitals.tsv | csvtk head -n 3 -H -t
4       5       6
1       2       3
7       8       0

$ export CSVTK_T=1; 
$ export CSVTK_H=true;
$ cat digitals.tsv | csvtk head -n 3
4       5       6
1       2       3
7       8       0
```

**Comment by shenwei356** [comment]
Available in [v0.13.0](https://github.com/shenwei356/csvtk/releases/tag/v0.13.0) and later versions
